src/panels/lovelace/hui-view.ts

```diff
@@ -264,7 +264,9 @@ export class HUIView extends LitElement {
     }
 
     const elements: HUIView["_badges"] = [];
-    const badges = processConfigEntities(config.badges);
+    const badges = processConfigEntities(
+      config.badges.filter((badge) => badge !== null)
```

Comment:
We should instead make sure we never store `null`.

Reply:
That's not possible for yaml

Reply:
Oh. Add a comment please. And maybe change the filter to just test for truthfulness:
```suggestion
      config.badges.filter(Boolean)
```

Reply:
I'll create a separate issue to not allow saving null in storage mode